airbyte-webapp/src/views/Connector/ServiceForm/types.ts

```diff
@@ -1,3 +1,5 @@
+// TODO: This needs to be converted to interface, but has int he current state a problem with index signatures
```

Comment:
:information_source: Converting this to `interface` fails at the moment, because of a missing index signature. I would like to fix that in a separate PR, to keep this PR only to the autofixed changes.